nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. it can be specified a later time for the message to be delivered. In Version 5 and below this can be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In Version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
 
 NOTE: Only send operations can be deferred. You cannot defer a publish or a reply. 
 
+## Difference between versions
+### Defer Handling
+In Versions 3-5 by calling `bus.Defer(...)` the handling of the message is deferred by certain amount of time. After that time the message is passed to be handled by the same endpoint, it means that endpoint need to have handler for deferred message to be able to handle it. Using this mechanism delayed delivery can be achieved by calling `bus.Defer(...)` and a having handler in which the actual `bus.Send(...)` will be called after elapsed amount of time. 
```

Comment:
The reason why I wrote it is that defer is actually sending the message to the same endpoint just like SendLocal. So if someone believe (and a lot of people do, even in Particular) that Defer actually sends the message to the Reciever after certain time. And it will not. It will send it to the same endpoint that called Defer. So if you want to actually send the message, you need a handler in which you will send it. On a reciever end you need to have an endpoint (to recieve it). So the ultimate different is that with Defer you might end up having 2 handlers: one on sender end and a second one on reciever end. I don't know what is the best way to ilustrate it, I guess sample would be the best.